src/utils/helpers/focus-trap/focus-trap.spec.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from 'react';
+import { mount } from 'enzyme';
+import FocusTrap from './focus-trap';
+
+// eslint-disable-next-line
```

Comment:
is this needed here?

Reply:
Yes, I didn't want to show eslint error about propTypes. This is a component created just for the test case